KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -249,6 +272,46 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         showCheckoutView(true)
+        setEarnLoyaltyForShowcase()
```

Comment:
You can wrap this logic with some kill switch in the configuration, so it won't appear if we had to make a hotfix release.

Reply:
Until the implementation of feature flags, if we need to make a hotfix we need to:
1. comment out this line so we don't show the configure can_earn and can_burn alerts on viewDidAppear
2. set the canEarn and canBurn to false. They can be found just before the setEarnLoyaltyForShowcase implementation (lines 285 and 286). This will hide the loyalty view entirely.

Reply:
Considering the feature flag task is the next to do in this sprint, we can merge.